tests/js/classes/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -150,6 +150,31 @@ class A4 {
 ================================================================================
 `;
 
+exports[`leading-line-break.js format 1`] = `
+====================================options=====================================
+parsers: ["babel", "flow", "typescript"]
+printWidth: 80
+                                                                                | printWidth
+=====================================input======================================
+class Demo {
+
+  firstMethod() {}
+  secondMethod() {}
+
+  thirdMethod() {}
+}
+
+=====================================output=====================================
+class Demo {
+  firstMethod() {}
```

Comment:
Something is not right here. When I do `./bin/prettier.js tests/js/classes/leading-line-break.js` from the command line, it outputs the following code as expected:

```js
class Demo {

  firstMethod() {}
  secondMethod() {}

  thirdMethod() {}
}
```

Reply:
> When I do `./bin/prettier.js tests/js/classes/leading-line-break.js` from the command line, it outputs the following code
> [...]

My bad, that is because Prettier was ignoring the formatting in this file. I did find and fix an issue with the formatting, should be OK now.